webapp/package-lock.json

```diff
@@ -3328,7 +3328,7 @@
         },
         "aproba": {
             "version": "1.2.0",
-            "resolved": "https://nexus3.itkweb.fr/repository/npm/aproba/-/aproba-1.2.0.tgz",
```

Comment:
Most of these changes are just changing the repository host from nexus3 to npmjs, while the package version of the dependency stays the same. I wonder if we should be preferring npmjs over this repository host in general.

Reply:
Oops sorry, it's the repository of my company :scream: please change it! :pray: for no one see that :p

Reply:
If we merge as is and move to the `npmjs` repository host, are there any actionable changes needed before merging?

Reply:
No I don't think so, because package-lock has `"integrity": "sha512-Y9J6ZjXtoYh8RnXVCMOU/ttDmk1aBjunq9vO0ta5x85WDQiQfUF9sIPBITdbiiIVcBo03Hi3jMxigBtsddlXRw==",` to be sure the files downloaded are the same. Just the source of this files will change. So if after this change a `npm install` from scratch (`rm -rf node_modules && npm cache clean`) work all is ok ;)

Reply:
@jfrerich To remove the rest of the bad repositories, maybe you can delete the `package-lock.json` and re-run `npm install` and `npm audit fix`. That might remove them.

Reply:
@crspeller, I performed the following actions on the latest commit.
* rm -rf node_modules/
* rm package-lock.json
* npm install
* npm audit fix 

Note the audit fix did not make any changes to `package-lock.json`

There are no longer any references to `nexus3` hosts.

Reply:
:pray: thank you very much :pray: